tools/enum_default_generator/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
The `tools/` pattern is fairly common in Go, but not as much in Rust.

Common build tool patterns in Rust include [the xtask pattern](https://docs.rs/xtasks/latest/xtasks/) and [build scripts](https://doc.rust-lang.org/cargo/reference/build-scripts.html).

While `build.rs` build scripts are common for code generation, that's usually in the context of being included as a 3rd party library and there being some special build options. We are just a predominantly generated library, so I'm inclined to say let's just start using the xtask pattern.

In general I think I would like to see the `update.sh` script end up as an xtask in time. Not something you need to add as an extra burden on yourself, but just something to think about when laying out the xtask code.